### PR TITLE
[WIP] Test PR for making Row/ColDefinition styleable.

### PR DESCRIPTION
**Pull request by jmacato**
Just testing stuff out and im getting asserts on ControlCatalog... not sure how it happens though.

### CODE REVIEW

File: samples/ControlCatalog.NetCore/ControlCatalog.NetCore.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
```
Comment (MarchingCube) [inline samples/ControlCatalog.NetCore/ControlCatalog.NetCore.csproj]: I think changing our projects to target .Net Core 3.1 should be a separate PR. We also might need to consider targeting .Net Standard 2.1 in our libraries.

Comment (jmacato) [reply samples/ControlCatalog.NetCore/ControlCatalog.NetCore.csproj]: Yeah, this is just a scratch/test PR (not gonna be merged as-is) so please expect the mess :) And control catalog is not working on my system without changing it to 3.1

**Comment by jmacato** [comment]
Didnt work, will try again sometime
